substrate/frame/revive/fixtures/build.rs

```diff
@@ -84,14 +83,21 @@ fn create_cargo_toml<'a>(
 	output_dir: &Path,
 ) -> Result<()> {
 	let mut cargo_toml: toml::Value = toml::from_str(include_str!("./build/_Cargo.toml"))?;
-	let mut set_dep = |name, path| -> Result<()> {
-		cargo_toml["dependencies"][name]["path"] = toml::Value::String(
-			fixtures_dir.join(path).canonicalize()?.to_str().unwrap().to_string(),
+	let uapi_dep = cargo_toml["dependencies"]["uapi"].as_table_mut().unwrap();
+
+	let metadata = MetadataCommand::new().exec().expect("Failed to fetch cargo metadata");
+	let uapi_pkg = metadata.packages.iter().find(|pkg| pkg.name == "pallet-revive-uapi").unwrap();
+
+	if uapi_pkg.source.is_none() {
+		uapi_dep.insert(
+			"path".to_string(),
+			toml::Value::String(
+				fixtures_dir.join("../uapi").canonicalize()?.to_str().unwrap().to_string(),
+			),
```

Comment:
How is this supposed to work? Is it set depending on whether we are in a workspace or not?

Reply:
yeah, would need to test it out, to make sure it works with the release deployment pipeline